modules/seqexec/server/src/main/scala/seqexec/server/gmos/package.scala

```diff
@@ -8,6 +8,8 @@ import seqexec.engine.Result.PartialVal
 package gmos {
   // N&S Partials. TODO add more details about the current step
   case object NSStart extends PartialVal
+  case object NSRowsConfigure extends PartialVal
+  case object NSTCSNod extends PartialVal
```

Comment:
`case class NSTCSNod(stage: NodAndShuffleStage)` ? I would expect you will need that for the UI.

Reply:
See above